app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -26,7 +26,7 @@ class Api::Rhsm::CandlepinProxiesController < Api::V2::ApiController
     before_action :set_organization_id, :except => [:hypervisors_update, :async_hypervisors_update]
     before_action :find_hypervisor_organization, :only => [:async_hypervisors_update, :hypervisors_update]
 
-    before_action :check_content_type, :except => :async_hypervisors_update
+    before_action :check_media_type, :except => :async_hypervisors_update
```

Comment:
There was a deprecation warning from Foreman, so I fixed that too.

```
18:05:00 rails.1   | 2020-08-31T18:05:00 [W|app|c7dbac04] DEPRECATION WARNING: You are using a deprecated behavior, it will be removed in version 2.3, #check_content_type is renamed to #check_media_type (called from repackage_message at /home/vagrant/katello/app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb:35)
```